trigger_undp_kenya_s01_demog.py

```diff
@@ -0,0 +1,91 @@
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to the pipeline configuration json file")
+    parser.add_argument("avf_uuid_file_path", metavar="avf-uuid-file-path",
+                        help="Domain that the first workspace of Rapid Pro is running on")
+
+    args = parser.parse_args()
+
+    google_cloud_credentials_file_path = args.google_cloud_credentials_file_path
+    pipeline_configuration_file_path = args.pipeline_configuration_file_path
+    avf_uuid_file_path = args.avf_uuid_file_path
+
+    log.info("Loading Pipeline Configuration File...")
+    with open(pipeline_configuration_file_path) as f:
+        pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
+    Logger.set_project_name(pipeline_configuration.pipeline_name)
+    log.debug(f"Pipeline name is {pipeline_configuration.pipeline_name}")
+
+    log.info("Downloading Rapid Pro access token...")
+    rapid_pro_token = google_cloud_utils.download_blob_to_string(
+        google_cloud_credentials_file_path, rapid_pro_token_url).strip()
+
+    rapid_pro = RapidProClient(rapid_pro_domain, rapid_pro_token)
+
+    log.info("Downloading Firestore UUID Table credentials...")
+    firestore_uuid_table_credentials = json.loads(google_cloud_utils.download_blob_to_string(
+        google_cloud_credentials_file_path,
+        pipeline_configuration.phone_number_uuid_table.firebase_credentials_file_url
+    ))
+
+    phone_number_uuid_table = FirestoreUuidTable(
+        pipeline_configuration.phone_number_uuid_table.table_name,
+        firestore_uuid_table_credentials,
+        "avf-phone-uuid-"
+    )
+    log.info("Initialised the Firestore UUID table")
+
+    log.info(f"Loading the uuids that are safe to send to")
+    with open(avf_uuid_file_path) as f:
+        safe_uuids = json.load(f)
+    log.info(f"Loaded {len(safe_uuids)} uuids")
+
+    log.info(f"Re-identifying the uuids")
+    safe_numbers = phone_number_uuid_table.uuid_to_data_batch(safe_uuids).values()
+    safe_urns = {f"tel:+{number}" for number in safe_numbers}
+    log.info(f"Re-identified {len(safe_urns)} uuids")
+
+    log.info("Downloading the latest contacts fields from Rapid Pro")
+    contacts = rapid_pro.get_raw_contacts()
+    urn_to_contact = dict()
+    for c in contacts:
+        for urn in c.urns:
+            urn_to_contact[urn] = c
+
+    log.info(f"Filtering the urns who haven't received demogs before")
+    urns_to_send_to = {urn for urn in safe_urns
+                       if urn_to_contact[urn].fields[demogs_attempted_variable] is None
+                       and urn_to_contact[urn].created_on > isoparse("2020-10-07T00:00+00:00")}
```

Comment:
control this from the ProjectStartDate

Reply:
I'll add a comment explaining why this check exists (in short, it's because we didn't set demogs_attempted for people who had completed it last time, so without this the set of people we should be triggering to is dominated by people who replied to our firs ad)

Reply:
I'll also change this to the start date. I thought it needed to be different, but on further thought I see that using the project start date will work fine